paper/paper.bib

```diff
@@ -0,0 +1,92 @@
+@inproceedings{hewson_bayesian_2023,
+	title = {Bayesian {Machine} {Scientist} for {Model} {Discovery} in {Psychology}},
+	author = {Hewson, Joshua and Strittmatter, Younes and Marinescu, Ioana and Williams, Chad and Musslick, Sebastian},
+	year = {2023},
+        booktitle = {NeurIPS 2023 AI for Science Workshop},
+}
+
+@inproceedings{musslick_evaluation_2023,
+	title = {An evaluation of experimental sampling strategies for autonomous empirical research in cognitive science},
+	volume = {45},
+	author = {Musslick, Sebastian and Hewson, Joshua TS and Andrew, Benjamin W and Strittmatter, Younes and Williams, Chad C and Dang, George T and Dubova, Marina and Holland, John Gerrard},
+	year = {2023},
+	note = {Issue: 45},
+}
+
```

Comment:
The capitalization of the articles seems inconsistent. "Bayesian {Machine} {Scientist}..." uses the capitalization of the original, while "An evaluation of experimental sampling strategies..." is all lowercase.

Reply:
Should now be addressed.